PostHog/Classes/PHGPayloadManager.m

```diff
@@ -13,6 +13,16 @@
 static NSString *const PHGAnonymousIdKey = @"PHGAnonymousId";
 static NSString *const kPHGAnonymousIdFilename = @"posthog.anonymousId";
 
+static NSString *const PHGActiveFeatureFlags = @"PHGActiveFeatureFlags";
+static NSString *const kPHGActiveFeatureFlags = @"posthog.activeFeatureFlags";
+
+static NSString *const PHGEnabledFeatureFlags = @"PHGEnabledFeatureFlags";
+static NSString *const kPHGEnabledFeatureFlags = @"posthog.enabledFeatureFlags";
```

Comment:
Inline with some feedback on posthog-android I think we can get away with just storing one item https://github.com/PostHog/posthog-android/pull/11

Reply:
Seems weird thought that we're adding something here that we know we are going to never need... As far as I can see the `activeFeatureFlags` is purely a legacy thing for the frontend js. For new libraries we only need the dictionary version as we can always derive the boolean outcomes from that?

Reply:
Sorry, I meant follow up PR as a stacked PR so the changes would be easier to see. I just brought in those changes though so they're all here! This is cleaned up now